### DISCUSSION TITLE
Welcome to CausalPy Discussions!

### DISCUSSION THREAD

**Discussion by twiecki** [pymc-labs/CausalPy#248] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jpreszler** [comment]
Hi Everyone,

I'm Jason from near Seattle, Washington in the US. I'm a mathematician turned data scientist with a strong software engineering background but am very new to causal inference. I learned about CausalPy from @drbenvincent's talk at PyData Global 2022 and was attracted to the use of PyMC for causal inference (I've tried to use PyMC as often as I can over the past few years since migrating from R to Python).

I've also wanted to get involved in contributing to an open-source project for a while and some issues here seemed like a good way to contribute, learn more about causal inference, and work with PyMC. Since I'm very much a causal novice, I'm not going to be implementing complicated new experiements or modeling techniques without some collaboration, but I'm happy to help others either use the package or implement new features and use my software development experience to fix bugs and improve CausalPy. My first contribution was adding lots of docstrings and doctest functionality and now I'm looking into some other bugs and possibly enhancements.

If anyone has usage issues or wants some help implementing a causal technique they have experience with, I'm happy to help.

**Reply by NathanielF** [reply]
Great to meet you @jpreszler! Thanks for your work on CausalPy!

**Reply by drbenvincent** [reply]
Great to have you on board @jpreszler 👍🏻
